libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts

```diff
@@ -201,19 +200,13 @@ export const HealthInsuranceForm: Form = buildForm({
               title: m.phoneNumber,
               width: 'half',
               variant: 'tel',
-              disabled: true,
               format: '###-####',
               placeholder: '000-0000',
               defaultValue: (application: Application) =>
                 (application.externalData.userProfile?.data as {
                   mobilePhoneNumber?: string
                 })?.mobilePhoneNumber,
             }),
-            buildDescriptionField({
```

Comment:
Why did you delete this description?

Reply:
It tells the user that the (only) way to change the email/phone number is to go to the Service Portal and change it there.